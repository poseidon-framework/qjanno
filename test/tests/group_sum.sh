qhs -H -O "SELECT foo,sum(bar),COUNT(*) cnt FROM big.csv WHERE bar IS NOT NULL GROUP BY foo ORDER BY cnt DESC LIMIT 3"
