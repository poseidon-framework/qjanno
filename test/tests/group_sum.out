foo sum(bar) cnt
a0 8 4
b1 4 2
b0 1 1
